doc/admin.md

```diff
@@ -105,6 +105,7 @@ It configures the connection parameters to stablish a connection to the Context
     host: '192.168.56.101',
     port: '1026',
     ngsiVersion: 'ld',
+    valueType: 'valueType',
```

Comment:
Given that value type is only used by NGSI-LD and taking into account that other LD-related config use the "Ld" token in the name, maybe it shoulb be named `ldValueType`?

In that case, the env var could be renamed to `IOTA_CB_NGSILD_VALUE_TYPE`